packages/client/ui/react-ui/src/components/embed/v3/EmbeddedCheckoutV3IFrame.tsx

```diff
@@ -46,27 +47,30 @@ export function EmbeddedCheckoutV3IFrame(props: CrossmintEmbeddedCheckoutV3Props
     }, [iframeClient]);
 
     return (
-        <iframe
-            ref={ref}
-            src={embedV3Service.iframe.getUrl(props)}
-            id="crossmint-embedded-checkout.iframe"
-            role="crossmint-embedded-checkout.iframe"
-            allow="payment *"
-            style={{
-                boxShadow: "none",
-                border: "none",
-                padding: "0px",
-                width: "100%",
-                minWidth: "100%",
-                overflow: "hidden",
-                display: "block",
-                userSelect: "none",
-                transform: "translate(0px)",
-                opacity: "1",
-                transition: "ease 0s, opacity 0.4s ease 0.1s",
-                height: `${height}px`,
-                backgroundColor: "transparent",
-            }}
-        />
+        <>
+            <iframe
+                ref={ref}
+                src={embedV3Service.iframe.getUrl(props)}
+                id="crossmint-embedded-checkout.iframe"
+                role="crossmint-embedded-checkout.iframe"
+                allow="payment *"
+                style={{
+                    boxShadow: "none",
+                    border: "none",
+                    padding: "0px",
+                    width: "100%",
+                    minWidth: "100%",
+                    overflow: "hidden",
+                    display: "block",
+                    userSelect: "none",
+                    transform: "translate(0px)",
+                    opacity: "1",
+                    transition: "ease 0s, opacity 0.4s ease 0.1s",
+                    height: `${height}px`,
+                    backgroundColor: "transparent",
+                }}
+            />
+            <CryptoWalletConnectionHandler iframeClient={iframeClient} />
```

Comment:
do we want to render this, even if crypto payments are not selected?

Reply:
changed